Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +372,47 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
 The following sections describes on how to retrieve basic information of an iOS app, that might be useful during a penetration test.
 
--- ToDo Objection: https://github.com/OWASP/owasp-mstg/issues/1247
+##### Mobile Security Framework (MobSF)
+
+MobSF is a penetration testing framework that is capable of analysing IPA files and can be used before even installing the app on your testing device.
+
+Once you have MobSF up and running you can open it in your browser by navigating to <http://127.0.0.1:8000>. Just select the IPA you want to analyse and MobSF will start it's job. The bigger the app the longer it takes, but usually you should get some feedback within a few minutes.
+
+The result screen might first be overwhelming but will have a very good overview of the potential attack surface and has automated a few things for you. With MobSF you can now for example:
+
+- Download a class-dump, if the app was written in Objective-C; if it is written in Swift no classpdump can be created.
+- Have access to the Info.plist
+- Exceptions in the App Transport Security (ATS) configuration will be raised
+
+There is much more information provided that you should explore, that might be helpful for you.
+
+##### Objection
+
+Once you have installed the app, there is further information to explore, where tools like objection come in handy. In the following example Frida is running as server on a jailbroken device and the app iGoat is running in the foreground. To attach to a process in this scenario you need to use the flag `--gadget` with the process name. When using objection you can retrieve different kind of information, where `env` will show you all the directory information of iGoat.
```

Comment:
Frida is running as server on -> Frida is running ~as a server~ on 

(Seems obvious)

different kind of information -> different kinds of information


I would also add `frida-ps -Ua` to get the process name.

Reply:
done